MobileCenter/MobileCenterTests/MSCustomPropertiesTests.m

```diff
@@ -47,17 +47,31 @@ - (void)testKeyValidate {
   
   // Then
   assertThat([customProperties properties], hasCountOf(0));
-  
+
+  // Too long key.
+  // When
+  NSString *tooLongKey = [@"" stringByPaddingToLength:129 withString: @"a" startingAtIndex:0];
```

Comment:
I see them now :). But I don't see the one for the exact max property count validation.